src/copy.jl

```diff
@@ -137,3 +137,15 @@ function Base.copy(model::AbstractModel)
     new_model, _ = copy_model(model)
     return new_model
 end
+
+"""
+    deepcopy(_::AbstractModel)
+
+Calls error. Calling `deepcopy` over a JuMP model is not supported, nor
```

Comment:
I don't see a need to have a docstring. The error message should be reasonably descriptive, and if necessary we can have a comment above the error explaining the technical issue with copying solvers that we have C pointers to.

Reply:
Done.